build-tests/api-extractor-scenarios/etc/includeForgottenExports/api-extractor-scenarios.api.md

```diff
@@ -0,0 +1,38 @@
+## API Report File for "api-extractor-scenarios"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// Warning: (ae-unresolved-inheritdoc-reference) The @inheritDoc reference could not be resolved: The package "api-extractor-scenarios" does not have an export "ForgottenExport"
+//
```

Comment:
**TODO:** This is a bug that I need to fix. It looks like `@inheritDoc` (and probably also `@link`) references expect to resolve to exported symbols.

Reply:
I ended up deciding not to fix this, see the "Known bugs/missing features" section in the PR description.